test/e2e/pre_provisioning_test.go

```diff
@@ -219,6 +219,79 @@ var _ = ginkgo.Describe("Pre-Provisioned", func() {
 			test.Run(cs, ns)
 		})
 
+		ginkgo.It("should succeed when attaching a shared block volume to multiple pods [disk.csi.azure.com][shared disk]", func() {
+			skipIfUsingInTreeVolumePlugin()
+			skipIfOnAzureStackCloud()
+			skipIfTestingInWindowsCluster()
```

Comment:
@andyzhangx I couldn't get Windows nodes to mount a block volume. It fails in `NodeStageVolume` [here](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/3efc8e013670e9083a236689ab36b279c9bd1ef1/pkg/azuredisk/nodeserver.go#L282) because it tries to mount the device number at the target location instead of the device path. Is this supposed to be supported on Windows?

Reply:
The specific error I see in the pod events is:

```log
  Warning  FailedMapVolume         8s (x5 over 49s)      kubelet                  MapVolume.MapPodDevice failed for volume "pvc-09d59441-7b27-4770-9065-6683cce6a3eb" : rpc error: code = Internal desc = Could not mount "6" at "\\var\\lib\\kubelet\\plugins\\kubernetes.io\\csi\\volumeDevices\\publish\\pvc-09d59441-7b27-4770-9065-6683cce6a3eb\\c7d3da4f-e18d-44b0-b7c2-29cada397395": rpc error: code = Unknown desc = not an absolute Windows path: 6
```

The driver logs for `NodePublishVolume` are:

```log
I0126 19:09:25.663317    4628 utils.go:95] GRPC call: /csi.v1.Node/NodePublishVolume
I0126 19:09:25.663317    4628 utils.go:96] GRPC request: {"publish_context":{"LUN":"0"},"staging_target_path":"\\var\\lib\\kubelet\\plugins\\kubernetes.io\\csi\\volumeDevices\\staging\\pvc-09d59441-7b27-4770-9065-6683cce6a3eb","target_path":"\\var\\lib\\kubelet\\plugins\\kubernetes.io\\csi\\volumeDevices\\publish\\pvc-09d59441-7b27-4770-9065-6683cce6a3eb\\003c7058-2704-497d-9904-285b8a1b776e","volume_capability":{"AccessType":{"Block":{}},"access_mode":{"mode":5}},"volume_context":{"cachingMode":"None","csi.storage.k8s.io/pv/name":"pvc-09d59441-7b27-4770-9065-6683cce6a3eb","csi.storage.k8s.io/pvc/name":"pvc-azuredisk-shared","csi.storage.k8s.io/pvc/namespace":"default","maxShares":"3","requestedsizegib":"256","skuname":"Premium_ZRS","storage.kubernetes.io/csiProvisionerIdentity":"1643223681277-8081-disk.csi.azure.com"},"volume_id":"/subscriptions/d64ddb0c-7399-4529-a2b6-037b33265372/resourceGroups/edreed-k8s-win-rg/providers/Microsoft.Compute/disks/pvc-09d59441-7b27-4770-9065-6683cce6a3eb"}
I0126 19:09:29.715824    4628 nodeserver.go:265] NodePublishVolume [block]: found device path 6 with lun 0
I0126 19:09:29.717283    4628 nodeserver.go:615] ensureBlockTargetFile [block]: making target file \var\lib\kubelet\plugins\kubernetes.io\csi\volumeDevices\publish\pvc-09d59441-7b27-4770-9065-6683cce6a3eb\003c7058-2704-497d-9904-285b8a1b776e
I0126 19:09:29.718064    4628 nodeserver.go:281] NodePublishVolume: mounting 6 at \var\lib\kubelet\plugins\kubernetes.io\csi\volumeDevices\publish\pvc-09d59441-7b27-4770-9065-6683cce6a3eb\003c7058-2704-497d-9904-285b8a1b776e
E0126 19:09:29.718914    4628 utils.go:100] GRPC error: rpc error: code = Internal desc = Could not mount "6" at "\\var\\lib\\kubelet\\plugins\\kubernetes.io\\csi\\volumeDevices\\publish\\pvc-09d59441-7b27-4770-9065-6683cce6a3eb\\003c7058-2704-497d-9904-285b8a1b776e": rpc error: code = Unknown desc = not an absolute Windows path: 6
```

Reply:
raw block volume is not supported on Windows